client/my-sites/site-settings/site-owner-transfer/start-site-owner-transfer.tsx

```diff
@@ -62,6 +62,23 @@ const ListItem = styled.li( {
 	marginBottom: '0.25em',
 } );
 
+const DomainsWrapper = styled.div( {
+	backgroundColor: '#F6F7F7',
+	marginTop: '2em',
+	marginBottom: '2em',
+	paddingTop: '1em',
+	paddingBottom: '1em',
+} );
+
+const DomainsWrapperItem = styled.div( {
+	paddingLeft: '1em',
+	fontWeight: 500,
+} );
+
+const GridiconStyled = styled( Gridicon )( {
+	paddingRight: '10px',
```

Comment:
`vertical-align: middle;` should do the trick.

Reply:
This works for me.
Another option would be setting : `display: "flex"` to the parent: `DomainsWrapperItem`.

Reply:
thanks for the pointers!